src/main/java/org/bonitasoft/engine/connector/uipath/UIPathConnector.java

```diff
@@ -24,197 +26,213 @@
-                : response.body().get("result");
-    }
-
-    protected Map<String, String> createAuthenticationHeaders(String token) {
-        Map<String, String> headers = new HashMap<>();
-        headers.put(HEADER_AUTHORIZATION_NAME, buildTokenHeader(token));
-        if (isCloud()) {
-            headers.put(HEADER_TENANT_NAME, getTenantLogicalName());
-        }
-        return headers;
-    }
-
-    protected UIPathService createService() {
-        if (service == null) {
-            OkHttpClient client = null;
-            if (LOGGER.isDebugEnabled()) {
-                HttpLoggingInterceptor interceptor = new HttpLoggingInterceptor();
-                interceptor.setLevel(HttpLoggingInterceptor.Level.BODY);
-                client = new OkHttpClient.Builder().addInterceptor(interceptor).build();
-            }
-
-            Builder retrofitBuilder = new Retrofit.Builder()
-                    .addConverterFactory(new WrappedAttributeConverter(mapper))
-                    .addConverterFactory(JacksonConverterFactory.create())
-                    .baseUrl(getUrl());
-
-            if (client != null) {
-                retrofitBuilder.client(client);
-            }
-
-            service = retrofitBuilder.build().create(UIPathService.class);
-        }
-        return service;
-    }
-
-    protected Map<Object, Object> toMap(Object inputParameter) {
-        Map<Object, Object> result = new HashMap<>();
-        for (Object row : (Iterable) inputParameter) {
-            if (row instanceof List && ((List) row).size() == 2) {
-                result.put(((List<Object>) row).get(0), ((List<Object>) row).get(1));
-            }
-        }
-        return result;
-    }
-
-    private static String appendTraillingSlash(String url) {
-        return url.endsWith("/") ? url : url + "/";
-    }
-
-    String getTenant() {
-        return (String) getInputParameter(TENANT);
-    }
-
-    String getTenantLogicalName() {
-        return (String) getInputParameter(TENANT_LOGICAL_NAME);
-    }
-
-    String getUser() {
-        return (String) getInputParameter(USER);
-    }
-
-    String getPassword() {
-        return (String) getInputParameter(PASSWORD);
-    }
-
-    String getUrl() {
-        return appendTraillingSlash(isCloud()
-                ? String.format("%s/%s/%s", CLOUD_ORCHESTRATOR_BASE_URL, getAccountLogicalName(),
-                        getTenantLogicalName())
-                : (String) getInputParameter(URL));
-    }
-
-    boolean isCloud() {
-        return (boolean) getInputParameter(CLOUD);
-    }
-
-    String getAccountLogicalName() {
-        return (String) getInputParameter(ACCOUNT_LOGICAL_NAME);
-    }
-
-    String getClientId() {
-        return (String) getInputParameter(CLIENT_ID);
-    }
-
-    String getUserKey() {
-        return (String) getInputParameter(USER_KEY);
-    }
+	private static final Logger LOGGER = LoggerFactory.getLogger(UIPathConnector.class.getName());
```

Comment:
Please use space instead of tabs for indentation.

Reply:
this has been updated